hapi-proto/src/main/proto/ResponseCode.proto

```diff
@@ -207,7 +207,8 @@ enum ResponseCodeEnum {
   MISSING_TOKEN_NAME = 193; // Token Name is not provided
   TOKEN_NAME_TOO_LONG = 194; // Token name is too long
   INVALID_WIPING_AMOUNT = 195; // The provided wipe amount must not be negative, zero or bigger than the token holder balance
-  TOKEN_IS_IMMUTABlE = 196; // Token does not have Admin key set, thus update/delete transactions cannot be performed
+  TOKEN_IS_IMMUTABLE = 196; // Token does not have Admin key set, thus update/delete transactions cannot be performed
   TOKEN_ALREADY_ASSOCIATED_TO_ACCOUNT = 197; // An <tt>associateToken</tt> operation specified a token already associated to the account
   TRANSACTION_REQUIRES_ZERO_TOKEN_BALANCES = 198; // An attempted operation is invalid until all token balances for the target account are zero
+  ACCOUNT_IS_TREASURY = 199;  // A selected account is a treasury
```

Comment:
imo, a better documentation would be: 
`An attempted operation is invalid because the account is a treasury`